data-raw/projects/proposals/sarcastat.yaml

```diff
@@ -0,0 +1,28 @@
+submitted_on: '2025-03-11'
+full_name: Jie Zhang
+github_username: Jie_evezhang
+job_position: Postdoc researcher
+phd_supervisor: ''
+affiliation: Region Midt (through Steno or AUH)
+project_title: UK Biobank study on statins and muscle strength and volume
+project_description: Statins, widely used as lipid-lowering therapy for
+  cardiovascular prevention, have demonstrated significant benefits in reducing
+  CVD events and all-cause mortality. However, their impact on muscle health
+  remains debated. While statins might have muscle-related side effects such as
```

Comment:
@arborzhang you can link more strongly here to the role of muscle health and strength to prevention of diabetes and its co-morbidities.